@xen-orchestra/web/src/components/host/HostPifSidePanel.vue

```diff
@@ -0,0 +1,459 @@
+                <!--
+                <UiComplexIcon size="medium">
+                  <VtsIcon :icon="faNetworkWired" accent="current" />
+                  <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                </UiComplexIcon>
+                <a href="">{{ networkNameLabel }}</a>
+                -->
+                <span v-tooltip class="text-ellipsis">{{ network?.name_label }}</span>
+              </div>
+            </template>
+            <template v-if="network?.name_label" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(network?.name_label)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- DEVICE -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('device') }}
+            </template>
+            <template #value>
+              {{ pif.device }}
+            </template>
+            <template v-if="pif.device" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(pif.device)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- PIF STATUS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ isBond ? $t('bond-status') : $t('pif-status') }}
+            </template>
+            <template #value>
+              <VtsConnectionStatus :status />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- PHYSICAL INTERFACE STATUS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('physical-interface-status') }}
+            </template>
+            <template #value>
+              <VtsConnectionStatus :status="physicalInterfaceStatus" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- VLAN -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('vlan') }}
+            </template>
+            <template #value>
+              {{ pif.vlan === -1 ? $t('none') : pif.vlan }}
+            </template>
+            <template v-if="pif.vlan !== -1" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(pif.vlan))"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- TAGS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('tags') }}
+            </template>
+            <template #value>
+              <UiTagsList class="tags value">
```

Comment:
I don't think the class is necessary. It overrides the styles defined in the component:
```suggestion
              <UiTagsList class="value">
```
Since `network` can be undefined, I think we should have a `v-if` here.